README.md

```diff
@@ -1,3 +1,10 @@
+## Overview
+
+This repository contains one solution which contains four main packages
```

Comment:
I would use the word "project" here, not "package" (to not confuse it with nuget terminology)

Reply:
Good point. BTW you may try "suggestion" feature of github.